### ISSUE TITLE
about compose sdk

### ISSUE THREAD

**Issue by tanlei02** [hCaptcha/hcaptcha-android-sdk#160]
implementation 'com.github.hCaptcha.hcaptcha-android-sdk:sdk:4.0.0'   and  implementation 'com.github.hCaptcha.hcaptcha-android-sdk:compose-sdk:4.0.0'   not worked.  implementation 'com.github.hcaptcha:hcaptcha-android-sdk:4.0.0'  works, but no compose sdk

**Comment by Syrou** [comment]
Have the same problem, the 4.0.0 https://jitpack.io/com/github/hCaptcha/hcaptcha-android-sdk/4.0.0/build.log has issues, and it seems the artifact for compose-sdk is not working @DSergiu

**Comment by Syrou** [comment]
@tanlei02 Unless you already solved it, for now
` implementation("com.github.hCaptcha.hcaptcha-android-sdk:compose-sdk:main-SNAPSHOT")` works

**Comment by Syrou** [comment]
Issue now is that this is not reachable at various time during building. @CAMOBAP It might be time to start using sonatype for this instead.

**Comment by DSergiu** [comment]
`compose-sdk` was released under version `4.0.1`. Please let me know if it works for you.

```
implementation 'com.github.hCaptcha.hcaptcha-android-sdk:compose-sdk:4.0.1'
```

**Comment by Syrou** [comment]
Seems to work just fine! Thanks!
